.vscode/launch.json

```diff
@@ -19,8 +19,8 @@
 			"module": "poetry",
 			"args": [
 				"poe", "query",
-				"--root", "<path_to_ragtest_root_demo>",
-				"--method", "global",
+				"--root", "/Users/gaudy-microsoft/Desktop/christmas_carol-new-version",
```

Comment:
This is pointing to something in your local machine

Reply:
I reverted this back to the old setting.